content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -25,6 +25,8 @@ further_reading:
   - .NET Core 3.1
   - .NET Core 2.1
 
+- ARM64 processors are supported for .NET 5 on Linux only.
```

Comment:
We should probably clarify that this line about ARM64 only applies to automatic instrumentation.

Reply:
We support the .NET runtimes listed above on the following architectures:
- win-x86
- win-x64
- linux-x64
- linuxmusl-x64 (is this the correct moniker?)
- linux-arm64 (only on .NET 5, added in v1.27.0)

Reply:
Done in https://github.com/DataDog/documentation/pull/10719/commits/5aef47e11f7eaa740f46668bc89296b4e93e162e